contracts/payoutStrategy/DirectPayoutStrategy/DirectPayoutStrategyImplementation.sol

```diff
@@ -0,0 +1,246 @@
+  function updateRoundFeeAddress(address payable _newFeeAddress) external isRoundOperator roundHasNotEnded {
+    roundFeeAddress = _newFeeAddress;
+    emit RoundFeeAddressUpdated(roundFeeAddress);
+  }
+
+  /**
+   * @notice Invoked by a round operator to make signal that a pending application turns to IN REVIEW status.*
+   *
+   * @param _applicationIndex Application index
+   */
+  function setApplicationInReview(uint256 _applicationIndex) external isRoundOperator roundHasNotEnded {
+    if (!_isPendingRoundApplication(_applicationIndex)) revert DirectStrategy__setApplicationInReview_applicationInWrongStatus();
+    _inReviewApplications[_applicationIndex] = true;
+
+    emit ApplicationInReview(_applicationIndex, msg.sender);
+  }
+
+  /**
+   * @notice Invoked by a round operator to make direct payments of funds to a project application.
+   *
+   * @dev It can be used to pay from a given address using `ERC20.transferFrom`, or from
+   * the configured vault in which case the AllowanceModule should be set as a Safe Module on the Safe Multisig vault,
+   * and the caller as delegate on the AllowanceModule.
+   * Using `transferFrom` only allow to pay with ERC20 tokens, and requires the indicated vault previously approved this
+   * contract to use such ERC20 token on it behalf.
+   * This 2 options are handled by the `payment.vault` parameter, if it set to an address different from address(0) then
+   * the function will follow the `transferFrom` path.
+   *
+   * To complete the payment it is required for the project application to be on status ACCEPTED.
+   *
+   * @param _payment payment data
+   */
+  function payout(Payment calldata _payment) external nonReentrant isRoundOperator {
+    uint256 currentStatus = RoundImplementation(roundAddress).getApplicationStatus(_payment.applicationIndex);
+    if (currentStatus != uint256(ApplicationStatus.ACCEPTED)) revert DirectStrategy__payout_ApplicationNotAccepted();
+
+    address usedVault = vaultAddress;
+
+    // use transfer from
+    if (_payment.vault != address(0)) {
+      if (_payment.token == address(0)) revert DirectStrategy__payout_NativeTokenNotAllowed();
+      /// @dev erc20 transfer to grant address
+      // slither-disable-next-line arbitrary-send-erc20,reentrancy-events,
+      SafeERC20Upgradeable.safeTransferFrom(
+        IERC20Upgradeable(_payment.token),
+        _payment.vault,
+        _payment.grantAddress,
+        _payment.amount
+      );
+      usedVault = _payment.vault;
+    } else { // use Safe multisig vault
+      IAllowanceModule allowanceModule = IAllowanceModule(_payment.allowanceModule);
+      allowanceModule.executeAllowanceTransfer(
+          vaultAddress,
+          _payment.token,
+          payable(_payment.grantAddress),
+          uint96(_payment.amount),
+          address(0),
+          0,
+          msg.sender,
+          _payment.allowanceSignature
+      );
+    }
+
+    // TODO - implement round fee
```

Comment:
Ah we could likely skip this cause fee is set to 0 and I don't think we intend on having fee set on this version of all.
Might be worth talking with product